components/status/ErrorCatchWidget.vue

```diff
@@ -35,8 +35,8 @@
     ></textarea>
     <div class="div_status-error-catch-next-step-txt" :style="next_step_cssprops">
       <p>
-        Please turn the instrument off and wait 10 seconds <br />
-        before attempting to use again
+        Please turn the instrument off and disconnect the USB.<br />
```

Comment:
This should be `Please turn the instrument off, unplug from the PC, and then wait 10 seconds before attempting to use again.`